dev-python/gitana/gitana-0.2.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+#PYTHON_REQ_USE="xml"
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="SQL-based Git Repository Inspector"
+HOMEPAGE="https://github.com/SOM-Research/Gitana"
+SRC_URI="https://github.com/SOM-Research/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+KEYWORDS="~amd64 ~x86"
+
+# problem with https://github.com/SOM-Research/Gitana/issues/5
+# wait for release and update version of git-python as soon as possible
+
+DEPEND="dev-lang/python:2.7[tk]
+	dev-python/git-python:0[${PYTHON_USEDEP}]
+	dev-python/mysql-connector-python:0[${PYTHON_USEDEP}]
+	dev-python/networkx:0[${PYTHON_USEDEP}]
+	dev-python/pillow:0[tk,${PYTHON_USEDEP}]
+	dev-python/PyGithub:0[${PYTHON_USEDEP}]
+	dev-python/python-bugzilla:0[${PYTHON_USEDEP}]
+	dev-python/simplejson:0[${PYTHON_USEDEP}]
+	>=dev-vcs/git-1.9.4
+	>=virtual/mysql-5.6"
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/Gitana-${PV}"
+
+src_prepare() {
+	cp "${FILESDIR}/${PV}/init.py" "${S}/__init__.py" \
+		|| die "Could not copy '${FILESDIR}/${PV}/init__.py' to '${S}/__init.py'"
+	cp "${FILESDIR}/${PV}/setup.py" "${S}/setup.py" \
+		|| die "Could not copy '${FILESDIR}/${PV}/setup.py' to '${S}/setup.py'"
+	epatch "${FILESDIR}/${PV}/case_study.py.patch"
+	epatch "${FILESDIR}/${PV}/db2json_gui.py.patch"
+	epatch "${FILESDIR}/${PV}/git2db.py.patch"
+	epatch "${FILESDIR}/${PV}/git2db_gui.py.patch"
+	epatch "${FILESDIR}/${PV}/gitana.py.patch"
+	epatch "${FILESDIR}/${PV}/gitana_gui.py.patch"
+	epatch "${FILESDIR}/${PV}/gitquerier_gitpython.py.patch"
+	epatch "${FILESDIR}/${PV}/init_dbschema.py.patch"
+	epatch "${FILESDIR}/${PV}/updatedb_gui.py.patch"
+	epatch "${FILESDIR}/${PV}/updatedb.py.patch"
```

Comment:
maybe the wiki-page at https://devmanual.gentoo.org/ebuild-writing/functions/src_prepare/epatch/ should be updated to reflect the usage of PATCHES variable?